tests/test_authentication/test_authentication.py

```diff
@@ -19,4 +19,24 @@ def test_invalid_token_returns_error_message(self):
         response = self.app_test.post(
             '/mrm?query=' + office_mutation_query,
             headers=headers)
-        self.assertIn("invalid token", str(response.data))
+        self.assertIn("You are not authorized to perform this action", str(response.data)) # noqa E501
+
+    def test_no_token(self):
+        """
+        Test that an error message is returned when a user
+        doesn't have a token
+        """
+        response = self.app_test.post(
+            '/mrm?query=' + office_mutation_query)
+        self.assertIn("HTTP_AUTHORIZATION", str(response.data)) # noqa E501
+
+    def test_incorrect_url(self):
+        """
+        Test that an error message is returned when a user
+        has an invalid token
+        """
+        headers = {"Authorization": "Bearer" + " " + INVALID_TOKEN}  # noqa E501
+        response = self.app_test.post(
+            '/mm?query=' + office_mutation_query,
+            headers=headers)
+        self.assertIn("he requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again.", str(response.data)) # noqa E501
```

Comment:
I think you have typo here. I think it should be "the requested" not "he requested"

Reply:
this has been taken care of.